rules/macos/persistence_crontab_creation.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2022/04/25"
+maturity = "development"
```

still in dev? If not then

```suggestion
maturity = "production"
```